crates/bevy_mesh/src/primitives/dim2.rs

```diff
@@ -843,6 +888,15 @@ pub struct RectangleMeshBuilder {
     half_size: Vec2,
 }
 
+impl RectangleMeshBuilder {
+    /// Creates a new [`RectangleMeshBuilder`] from a full width and height.
+    pub const fn new(width: f32, height: f32) -> Self {
```

Comment:
Same potential consideration as the Rhombus builder.

Reply:
Same thing as rhombus: a negative width / height causes the mesh to disappear. Adding a `debug_assert!`!